README.md

```diff
@@ -101,6 +106,21 @@ sbatch scripts/run_short_slurm.sh # 8k-64k
 # for the API models, note that API results may vary due to the randomness in the API calls
 bash scripts/run_api.sh 
 ```
+### Run on Intel Gaudi
+If you want to enable the evaluation on vLLM with Intel Gaudi, you can use the following commands:
+```bash
+## Build vllm docker image
+cd scripts/vllm-gaudi
+bash build_image.sh
+
+## launch vllm container, change `LLM_MODEL_ID` and `NUM_CARDS` as your need
+cd scripts/vllm-gaudi
+bash launch_container.sh
+
+## evalute
+bash scripts/run_eval_vllm_gaudi.sh
```

Comment:
Before running this line, the user would need to `cd ..` back to the base directory where `eval.py` is.